scripts/devnet.yaml

```diff
@@ -1,5 +1,5 @@
-chainmaind-777:
-  cmd: ./build/ethermintd
+cronos-777:
```

Comment:
missing `d`?

Reply:
hmm, since it's chain id here, maybe we shouldn't add the `d`, I removed the `d` in the other chain-ids.